scripts/setup

```diff
@@ -6,5 +6,6 @@
 # CDS-ILS is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
-invenio setup --skip-pages --verbose
+invenio setup --skip-pages --skip-vocabularies --verbose
```

Comment:
are we missing the licenses from opendefinition by skipping it?

Reply:
no, we have moved the opendefinition already to the cds-ils command for vocabularies